.github/workflows/build-test-package.yml

```diff
@@ -256,16 +256,16 @@ jobs:
       run: |
         cat > git-configure-build-sem.sh << EOF
         #!/bin/sh
-        ln -s -f -T /ITKPythonPackage/ITK-cp37-cp37m-manylinux2014_x86_64 /work/ITK-cp37-cp37m-manylinux2014_x86_64
+        ln -s -f -T /ITKPythonPackage/ITK-cp39-cp39-manylinux2014_x64 /work/ITK-cp39-cp39-manylinux2014_x64
```

Comment:
Progress :-)

It now looks like it now cannot find the TBB install.

Maybe adding

```
ln -s -f -T /ITKPythonPackage/oneTBB-prefix /work/oneTBB-prefix
```

here will do it?

Reply:
@tbirdso good idea on centralization of support. We could add when it make sense, but I think most modules use a vanilla version of what is in ITKModuleTemplate.